### ISSUE TITLE
class \core_search\area\base_activity is deprecated

### ISSUE THREAD

**Issue by thepurpleblob** [streamingltd/MEDIAL-Moodle-Activity#4]
In mod/helixmedia/classes/search/activity.php

The base class should be \core_search\base_activity not \core_search\area\base_activity. 

See the upgrade notes in search/upgrade.php under the section for Moodle 3.2.

```
=== 3.2 ===

* Base search area classes have been renamed, please update your search areas to use the classes below:
  - \core_search\area\base has been renamed to \core_search\base
  - \core_search\area\base_mod has been renamed to \core_search\base_mod
  - \core_search\area\base_activity has been renamed to \core_search\base_activity
```

In future versions of Moodle, this will fail completely.

**Comment by dustinbrisebois** [comment]
This has been resolved as of Moodle 3.6 --- 

https://github.com/streamingltd/MEDIAL-Moodle-Activity/blob/master/classes/search/activity.php#L36

Is now: class activity extends \core_search\base_activity {

This plugin is not fully Moodle 3.6 compatible.

**Comment by tim1mw** [comment]
This is fixed in plugin version 2019052102.
